ext/standard/array.c

```diff
@@ -3869,15 +3875,15 @@ static zend_always_inline void php_array_merge_wrapper(INTERNAL_FUNCTION_PARAMET
 }
 /* }}} */
 
-/* {{{ proto array array_merge(array arr1 [, array ...])
+/* {{{ proto array array_merge([array arr1 [, array ...]])
```

Comment:
```suggestion
/* {{{ proto array array_merge([, array ...])
```

Reply:
I really was guessing on the correct syntax here... :)